main/zxing-cpp/template.py

```diff
@@ -20,3 +20,8 @@
 @subpackage("zxing-cpp-devel")
 def _(self):
     return self.default_devel()
+
+
+@subpackage("zxing")
```

Comment:
subpackage names should have the parent package name as a prefix (this will be enforced in the future)
```suggestion
@subpackage("zxing-cpp-progs")
```

Reply:
Done. Thanks.